index.html

```diff
@@ -167,6 +167,28 @@ <h1 class="about-title">About Myself</h1>
                 </ul>
             </section>
             </section>
+
+            <section class="last-section-mobile">
+                <h2>Contact me</h2>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+                    <form action="#" method="post">
+                        <fieldset class="mobile-contact">
+                            <label for="username">
+                                <input type="text" class="input" required name="username" maxlength="30" placeholder="Username">
+                            </label>
+                            <label for="email">
+                                <input type="email" class="input" required name="email" placeholder="name@domain.com">
+                            </label>
+                            <label for="message">
+                                <textarea name="message" class="message-mobile" required maxlength="500" placeholder="Write your message here"></textarea>
+                            </label>
+                            <div class="button-contact">
+                              <button type="submit" class="btn last-btn">Get in touch</button>
```

Comment:
Kindly add hover and pressed state for your submit button as mentioned on the project requirements.
<img width="288" alt="Screenshot 2023-01-18 at 12 56 26 AM" src="https://user-images.githubusercontent.com/25887253/213021379-b5d6b51e-2436-4033-ba75-c4fa880b08f9.png">

Reply:
Thanks for the review @huseyinbiyik